### ISSUE TITLE
gmx.at: detection

### ISSUE THREAD

**Issue by 10Meisterbaelle** [uBlockOrigin/uAssets#15000]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://www.gmx.at/fairplay/?section=fairplay`

### Category

detection

### Describe the issue

When you log into GMX, you only sometimes get to this website (this is intended by the website) which asks you to disable your adblocker and relogin. I confirmed this on gmx.at, but it probably also occurs on gmx.net, gmx.ch, gmx.de.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.44.4
Firefox: 105
filterset (summary): 
  network: 86087
  cosmetic: 43817
  scriptlet: 17858
  html: 752
listset (total-discarded, last updated): 
  default: 
    user-filters: 0-0, never
    ublock-filters: 33584-54, 2h.59m
    ublock-badware: 4209-0, 2h.58m
    ublock-privacy: 268-0, 2h.57m
    ublock-abuse: 76-0, 2h.56m
    ublock-unbreak: 1872-0, 2h.55m
    ublock-quick-fixes: 237-0, 7h.33m
    easylist: 63512-609, 4d.7h.11m
    easyprivacy: 30599-1714, 4d.7h.10m
    urlhaus-1: 7076-0, 7h.32m
    plowe-0: 3674-3, 8d.9h.20m
    DEU-0: 5947-2, 7h.31m
filterset (user): [empty]
trustedset: 
  added: [array of 1 redacted]
modifiedUserSettings: 
  cloudStorageEnabled: true
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 388 ms (selfie)
  maxAssetCacheWait: 136 ms
popupPanel: 
  blocked: 11
  blockedDetails: 
    gmx.net: 5
    ui-portal.de: 2
    uicdn.com: 3
    uimserv.net: 1
```
</details>

**Comment by dportvine** [comment]
This detection?

<details>
<summary>Gif</summary>

![1](https://user-images.githubusercontent.com/111344219/191908051-95becaa1-1737-4e90-b8d4-a7f99d5d2963.gif)
</details>

**Comment by okiehsch** [comment]
Is there any way to reproduce this consistently?
I could reproduce the first time I tried but since then no matter what I did, I can't reproduce anymore.

**Comment by okiehsch** [comment]
If it works add
`gmx.*##+js(set, adBlockDetectionResult, undefined)`
to cover all sites, I could reproduce using `gmx.net`.

**Comment by okiehsch** [comment]
> Also no account/login is required.

Hm, I don't get any anti adblock message even if  I use Firefox, maybe it depends on the geo location.

**Comment by 10Meisterbaelle** [comment]
> Is there any way to reproduce this consistently?
> 
> I could reproduce the first time I tried but since then no matter what I did, I can't reproduce anymore.

Yes, if you click on the link (https://www.gmx.at/fairplay/?section=fairplay), it will consistently show whether it detected an ad blocker or not without having to log in and hope it comes up.

**Comment by okiehsch** [comment]
@10Meisterbaelle thx, so 
`gmx.*##+js(set, adBlockDetectionResult, undefined)`
should work for all sites.

**Comment by 10Meisterbaelle** [comment]
> gmx.at##+js(set, adBlockDetectionResult, undefined)



> If it works add `gmx.*##+js(set, adBlockDetectionResult, undefined)` to cover all sites, I could reproduce using `gmx.net`.



> I can reproduce consistently using Firefox
> 
> @10Meisterbaelle Add this to "My filters" and test:
> 
> `gmx.at##+js(set, adBlockDetectionResult, undefined)`
> 
> Works for me.

I cannot confirm whether this works or not yet, because it doesn't show I have no adblock, it shows nothing. If this reappears when logging in, I will comment.

**Comment by okiehsch** [comment]
No, those filters were necessary at the time  because
`3p,domain=example.*`
was not yet supported.

**Comment by okiehsch** [comment]
> more than these 3 domains, then should I convert those into gmx.* too?

Sure, why not?

**Comment by 10Meisterbaelle** [comment]
This patch did not fix the problem when logging in, the issue reoccured. Please reopen this issue.

**Comment by okiehsch** [comment]
I got redirected one time to 
![image](https://user-images.githubusercontent.com/16838044/194066568-ffc8f0ff-14fa-4e37-b98b-6b71750d3289.png)

Now I can't reproduce anymore, but my screenshot is not an anti adblock warning.

**Comment by 10Meisterbaelle** [comment]
> I got redirected one time to ![image](https://user-images.githubusercontent.com/16838044/194066568-ffc8f0ff-14fa-4e37-b98b-6b71750d3289.png)
> 
> Now I can't reproduce anymore, but my screenshot is not an anti adblock warning.

This is what I get too, even though it is no anti adblock warning, it still redirects you to the page that is supposed to tell you to disable your adblocker, which is hidden by uBlock Origin, and won't let you in your account until you relogin.
